benchmarks/fs2/src/main/scala/laserdisc/fs2/RedisClientBench.scala

```diff
@@ -0,0 +1,70 @@
+package laserdisc.fs2
+
+import cats.effect._
+import cats.implicits._
+import laserdisc._
+import laserdisc.all._
+import laserdisc.auto._
+import log.effect.LogWriter
+import log.effect.fs2.SyncLogWriter
+import redis.clients.jedis.Jedis
+import org.openjdk.jmh.annotations._
+
+import scala.concurrent.ExecutionContext
+
+object shared {
+  implicit def contextShift: ContextShift[IO] = IO.contextShift(ExecutionContext.global)
```

Comment:
ExecutionContext.global uses a FJP internally (i.e. work-stealing) and this may not be what you'd want. Also, `val` so we don't recreate over and over

Reply:
Both good suggestions - I'll use a fixed pool and make it a `val`